### ISSUE TITLE
import_local does not work with app installed via an AppConfig

### ISSUE THREAD

**Issue by sdementen** [evenicoulddoit/django-rest-framework-serializer-extensions#38]
if you specify a string in a serializer expandable_fields, the `import_local `(https://github.com/evenicoulddoit/django-rest-framework-serializer-extensions/blob/master/rest_framework_serializer_extensions/utils.py#L8) is called which reimplents functionally the `import_string` (https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.module_loading.import_string) function but does not handle properly an 
```
INSTALLED_APPS = [
...
"myapp.apps.MyAppConfig",
...
]
```

I would suggest to use `import_string `directly (django >= 1.7)

**Comment by evenicoulddoit** [comment]
Hi @sdementen, thanks for the report. I'm just trying to understand the problem a little more, I presume you're using a string reference to either a serializer or a HashIds instance?

I think the original intention of the `import_string` method, was not to re-invent the wheel, but to provide a little more safety than is ultimately offered by just using `import_module` (which is what django does). In theory, importing a module could have side-effects, that I was trying to guard against here. That said, this is all outside of user space, and so perhaps that assumption was a bit too harsh.

I'll have a think, but perhaps your suggestion of just using `import_string` from Django is the best idea :)

**Comment by sdementen** [comment]
Without the change, I get the exception `Cannot import from outside installed apps`  from
```
        if '.'.join(path_pieces) not in settings.INSTALLED_APPS:
            path_pieces.pop()
        else:
            break
    if not path_pieces:
        raise AssertionError(
            "Cannot import from outside installed apps" ...
        )
    return getattr(importlib.import_module(path), name)
```

You are using INSTALLED_APPS to test some path is within some app. This assumes the apps added in the INSTALLED_APPS are the name of a package/folder which is not always the case as it can point to an application configuration class (https://docs.djangoproject.com/en/3.0/ref/settings/#installed-apps).

**Comment by evenicoulddoit** [comment]
Hi @sdementen - a fix, which removes the use of `import_local()`, thus allowing this to work, was just included in 2.0.1 

Let me know if you have any further issues. Ian

**Comment by sdementen** [comment]
Great, tx @evenicoulddoit for this fix!
